tests/influxdb/client_test.py

```diff
@@ -29,6 +29,9 @@
 from influxdb import InfluxDBClient, InfluxDBClusterClient
 from influxdb.client import InfluxDBServerError
 
+DSN_STRING = 'influxdb://uSr:pWd@host:1886/db'
+CLUSTER_DSN_STRING = 'influxdb://uSr:pWd@host1:8086,uSr:pWd@host2:8086/db'
```

Comment:
Can you declare this in a set_up instead? I prefer not to have class-level attributes